packages/sdk/client-services/src/packlets/invitations/invitations-handler.ts

```diff
@@ -118,10 +132,8 @@ export class SpaceInvitationsHandler extends AbstractInvitationsHandler<DataSpac
 
           return {
             spaceKey:
-              type === Invitation.Type.INTERACTIVE_TESTING || type === Invitation.Type.MULTIUSE_TESTING
-                ? space.key
-                : undefined,
-            authMethod: invitation.authMethod!
+              authMethod === Invitation.AuthMethod.NONE ? operations.getInvitationContext().spaceKey : undefined,
```

Comment:
seems like getInvitationContext was already called above, and placed into the invitation object, can we just grab space key from there?

Reply:
also TODO: to discuss whether we should share space key (and name) before auth

Reply:
I was thinking that sharing the spacekey/name before auth should be optional (it could be a security setting on the space)

Reply:
let's make it optional in the impl, and address later how the app does this; e.g., could be on by default if you already know the peer's key